ntbs-service-unit-tests/DataAccess/DataQualityRepositoryTests.cs

```diff
@@ -303,6 +303,44 @@ public async Task FiltersOutNotificationsWithTheAlertTypeAlreadyPresent()
             Assert.Empty(notificationIds);
         }
 
+        public static IEnumerable<object[]> GetData_TBServiceReferralDate_CorrectlyTriggersDataQualityAlert()
```

Comment:
I'm not sure I've seen this way of getting test data in this project - this is workaround for not being to use DateTime.Parse for InLineData.

Reply:
I think what you want is TheoryData: https://github.com/publichealthengland/ntbs_Beta/blob/0c70ee0498b035678ec5c1599e754ac1dde382ed/ntbs-service-unit-tests/Models/Entities/NotificationTest.cs#L37

Reply:
Thanks!